update_prevalence.py

```diff
@@ -1122,6 +1149,11 @@ def parse_csv(cache: DataCache, model: Type[Model], url: str) -> List[Model]:
     reader = csv.reader(lines)
     fields = [re.sub("^7", "Seven", re.sub("[^A-Za-z0-9_]", "_", name)) for name in next(reader)]
 
+    # verify we have every field we expect
+    for field_name in model.__fields__:
+        if field_name not in fields:
+            raise ValueError(f"Did not find expected '{field_name}' column in header of {url}")
```

Comment:
If a column which is marked Optional in our model, the source could slightly rename the column and we'd silently start treating the entire column as None.  This block makes us more sensitive to those sorts of structural changes.